### DISCUSSION TITLE
【读书会】《小狗钱钱》 - 第1期 | ApacheCN

### DISCUSSION THREAD

**Discussion by yanqiang921106** [apachecn/home#231] [Ideas]
# <center>【读书会】《小狗钱钱》 - 第1期 | ApacheCN<center>

* 【推荐】读书会: 《小狗钱钱》博多·费舍尔
* 【推荐】留言地址:  <https://github.com/apachecn/financial/discussions/4>

> 参与

* QQ群【临时群】: 1062944139
* 负责人紫色木槿: 1076003661 (QQ)

> 时间

**每两周三晚8点，月初会发布本月阅读书籍2-4本，顺序阅读。

* 时间周期: 2021-06-02 ～ 2021-06-16 （共 14 天，对自己严格的建议控制在7天内）
* 语音频率: 每两周三晚9点语音一次，周末休息跳过
* **群友发言：**5~10人，5分钟/人，计划时长0.5-1小时	
* **[@紫色木槿](https://github.com/yanqiang921106)** 会进行语音: 沟通进度和督促学习

> 内容

1. 分享读书心得
2. 学到的知识
3. 可以应用到实际工作生活中的
4. 自己的应用计划			

每一本书，专门一个issues，供大家留言分享		

> 活动目的

1. 培养自己对理财的正确认识
2. 选出2个人作为大肥羊学校的学员，作为其他活动的代理班长		

---

* 总结：一人可以走的很快，但是一群人却可以走的很远

**Comment by yanqiang921106** [comment]
关于挣钱:
1.为别人解决一个难题，那么你就能赚到许多钱；
2.把精力集中在你知道的、能做的和拥有的东西上。而不是浪费在不知道的、不能做的事情上。遇事不要为自己找借口。
3.无论在什么时候都不能把希望只寄托在一份工作上，它持续的时间不会像你设想的那么长，所以你要立即寻找另一份替代的工作。
4.情况顺利的时候，人人都能挣到钱。只有在逆境中，一切才能见分晓。

**Comment by yanqiang921106** [comment]
1、你是否能挣到钱，最关键的并不是你有没有好点子，也不是你有多聪明，而是你的自信程度。

**Comment by yanqiang921106** [comment]
学习就是认识新观念和新想法的过程。假如人们始终以同一种思维方式来考虑问题的话，那么始终只会得到同样的结果。因为我对你讲述的许多内容是你以前从未接触过的，所以我建议你，在你还没有做之前，不要轻易下结论。没有想象力的人是很难成就大事的。我们对一件事投入的精力越多，成功的可能性也越大。可是大多数人把精力放在自己并不喜欢的事情上，而不去想象自己希望得到的东西。

**Comment by now-101** [comment]
1、毁掉信用卡
2、在许可的范围内按最低的分期付款数目标准支付
3、对于消费贷款，贷款的人应当遵守的一个原则，就是将不用于生活的那部分钱的一半存下来，另一半用于偿还贷款。

**Comment by yanqiang921106** [comment]
挑选基金时的注意事项：
1．基金应该至少有10年历史。假如它在这么长时间内一直有丰厚的利润，那我们可以认为，未来它也会运作良好。
2．应该选择大型的跨国股票基金。这种基金在世界各地购买股票，以此分散风险，所以十分安全。
3．对基金的走势图进行比较。我们应该观察在过去10年间哪些基金的年终利润最好。

**Comment by yanqiang921106** [comment]
我们推崇一种聪明的、简朴的生活方式。也就是说，宁愿购买一件一流产品，也不要不停地买许多的二流产品。而且，不要仅仅因为一件产品的外观不再时髦而新产品正在流行，就不断追逐新鲜的东西。
生活质量不是由越来越多的高科技产品堆砌而成的，而体现在一些别的方面，比如悠闲地享受一下生活，增进邻里关系，表达感情或者从事艺术性和创造性的活动。

**Comment by jiangzhonglian** [comment]
1. 梦想储蓄罐
2. 成功日记
3. 减少没必要的消费，将多余的钱用于理财投资
4. 如果有消费贷款，那么一半用于理财投资，一半用于偿还消费贷

道理很简单，就看你怎么执行力（根据风险偏好，慢慢修正）

![image](https://user-images.githubusercontent.com/9199175/121164581-4098ee00-c882-11eb-8f59-2ae03cdbad89.png)

**Comment by liukai1919** [comment]
1.有一个愿望——赚钱的动力
2.下定决心，做或者不做
3.自信
4.利用兴趣挣钱
5.水滴石穿，每天做一点点
6.不要借消费贷款
7.拥有生产资料
8.机会只留给有准备的人
9.向积极的方向思考
10.给自己一点压力（动力？）

**Comment by now-101** [comment]
金先生说："越是把注意力放在疼痛上，我就越会觉得疼。谈论疼痛就像给植物施肥一样。所以我很多年以前就改掉了抱怨的习惯"

**Comment by now-101** [comment]
金钱就像一个放大镜，它帮你更充分地展现出你本来的样子。好人可以用钱做很多好事。而如果你是盗贼，那你很可能会把钱挥霍在一些蠢事上。

**Comment by now-101** [comment]
要想过更幸福、更满意的生活，人就得改变自身。这和钱无关，金钱本身既不会使人幸福，也不会带来不幸。金钱是中性的，既不好，也不坏。只有当钱属于某一个人的时候，它才会对这个人产生好的影响或者坏的影响。钱可以被用于好的用途，也可以被用于坏的用途。一个幸福的人有了钱会更幸福；而一个悲观忧虑的人，钱越多，烦恼就越多。

**Comment by now-101** [comment]
1．确定自己希望获得财务上的成功。2．自信，有想法，做自己喜欢做的事。3．把钱分成日常开销、梦想目标和金鹅账户三部分。4．进行明智的投资。5．享受生活。

**Comment by now-101** [comment]
1．确定自己希望获得财务上的成功。2．自信，有想法，做自己喜欢做的事。3．把钱分成日常开销、梦想目标和金鹅账户三部分。4．进行明智的投资。5．享受生活。

**Comment by now-101** [comment]
投资的三个原则：
1．应该把钱投资在安全的地方。
2．我的钱应该下很多“金蛋”。
3．我们的投资应该简单明了。

**Comment by jiangzhonglian** [comment]
1
